portfolio/src/main/webapp/index.html

```diff
@@ -8,8 +8,15 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
+      <h1>Grace's Website</h1>
+      <p>My name is Grace. I'm a STEP intern. Welcome to my website!</p>
+        <img src="images/propic.JPG" width="300">
+      <p>
+            Learn more about my random  <a href="moments.html">moments</a> 
```

Comment:
nit: inconsistent indentation

Reply:
is it the img tag that is inconsistent or the multi line indent in the last <p> that is inconsistent?

Reply:
img tag is indented differently than its sibling element. If your code editor keeps doing this on its own you may need to self-close the img tag like `<img ... />`